src/test/kotlin/no/nav/familie/tilbake/beregning/TilbakekrevingsberegningVilkårTest.kt

```diff
@@ -43,385 +48,555 @@ class TilbakekrevingsberegningVilkårTest {
+                        periode = vilkårsvurdering.periode,
+                        tilbakekrevingsbeløp = tilbakekrevingsbeløp,
+                        skatteprosent = skatteprosent
+                    )
+                )
 
-    @Test
-    fun `beregn skal kreve tilbake deler ved grov uaktsomhet ved når særlige grunner og ikke ilegge renter når det er false`() {
-        vurdering = vurdering.copy(
-            aktsomhet = VilkårsvurderingAktsomhet(
-                aktsomhet = Aktsomhet.GROV_UAKTSOMHET,
-                begrunnelse = "foo",
-                særligeGrunnerTilReduksjon = true,
-                ileggRenter = false,
-                andelTilbakekreves = BigDecimal.valueOf(70)
+            val resultat = beregn(
+                vilkårVurdering = vilkårsvurdering,
+                feilutbetalt = tilbakekrevingsbeløp,
+                perioderMedSkatteprosent = grunnlagPeriodeMedSkattProsent,
+                beregnRenter = true,
+                bruk6desimalerISkatteberegning = false
+            )
+            val resultatMed6Desimaler = beregn(
+                vilkårVurdering = vilkårsvurdering,
+                feilutbetalt = tilbakekrevingsbeløp,
+                perioderMedSkatteprosent = grunnlagPeriodeMedSkattProsent,
+                beregnRenter = true,
+                bruk6desimalerISkatteberegning = true
             )
-        )
 
-        // assert
-        val resultat: Beregningsresultatsperiode =
-            beregn(vurdering, BigDecimal.valueOf(10000), Lists.newArrayList(grunnlagsperiodeMedSkatteprosent), true)
-        resultat.tilbakekrevingsbeløp shouldBe BigDecimal.valueOf(7000)
-        resultat.renteprosent shouldBe null
-        resultat.rentebeløp.shouldBeZero()
+            validerResultat(
+                resultat = resultat,
+                vilkårsvurdering = vilkårsvurdering,
+                vurdering = Aktsomhet.FORSETT,
+                feilutbetalt = tilbakekrevingsbeløp,
+                renteprosent = RENTEPROSENT,
+                rentebeløp = BigDecimal.valueOf(421),
+                tilbakekrevingsbeløpUtenRenter = tilbakekrevingsbeløp,
+                tilbakekrevingsbeløp = BigDecimal.valueOf(4633),
+                tilbakekrevingsbeløpEtterSkatt = BigDecimal.valueOf(3201),
+                skattebeløp = BigDecimal.valueOf(1432)
+            )
+
+            validerResultat(
+                resultat = resultatMed6Desimaler,
+                vilkårsvurdering = vilkårsvurdering,
+                vurdering = Aktsomhet.FORSETT,
+                feilutbetalt = tilbakekrevingsbeløp,
+                renteprosent = RENTEPROSENT,
+                rentebeløp = BigDecimal.valueOf(421),
+                tilbakekrevingsbeløpUtenRenter = tilbakekrevingsbeløp,
+                tilbakekrevingsbeløp = BigDecimal.valueOf(4633),
+                tilbakekrevingsbeløpEtterSkatt = BigDecimal.valueOf(3202),
+                skattebeløp = BigDecimal.valueOf(1431)
+            )
+        }
```

Comment:
Kanskje litt smak og behag, men jeg liker ofte at man asserter _en_ ting i en test. Tidligere var det riktignok to ting 😝 

resultat.skattebeløp shouldBe BigDecimal.valueOf(1432)
resultatMed6Desimaler.skattebeløp shouldBe BigDecimal.valueOf(1431)

Da blir det veldig lett å lese hva man tester. Alternativt kan man kanskje lage flere tester som sjekker andre ting? Ikke alltid praktisk å gjøre det på denne måten. Hva som blir riktig her kan godt diskuteres. Jeg kan fint leve med dette også :)